### PR TITLE
Add userCameraCap (BBB 2.4.5)

### PR DESCRIPTION
**Pull request by SamuelWei**
Implements #112

### CODE REVIEW

**Comment by SamuelWei** [comment]
@sualko After or before implementing this, we can think about releasing a new minor version (see my release draft).
As #109 is not effecting the feature support of the api, we could independently make the release decision.

As soon as BBB 2.5 is released we also have to make some changes to this library. I'll create issues and PR soon.

File: src/Parameters/CreateMeetingParameters.php [hunk 1]
```diff
@@ -528,6 +535,24 @@ public function isBreakout()
         return $this->isBreakout;
     }
 
+    /**
+     * @return bool
+     */
+    public function isUserCameraCapDisabled()
```
Comment (FelixJacobi) [inline src/Parameters/CreateMeetingParameters.php:541]:
```suggestion
    public function isUserCameraCapDisabled(): bool
```

File: src/Parameters/CreateMeetingParameters.php [hunk 2]
```diff
@@ -528,6 +535,24 @@ public function isBreakout()
...
+    {
+        return $this->userCameraCap === 0;
+    }
+
+    /**
+     * @return CreateMeetingParameters
+     */
+    public function disableUserCameraCap()
```
Comment (FelixJacobi) [inline src/Parameters/CreateMeetingParameters.php:549]:
```suggestion
    public function disableUserCameraCap(): self
```

File: tests/unit/Parameters/CreateMeetingParametersTest.php [hunk 3]
```diff
@@ -157,4 +157,16 @@ public function testGetPresentationsAsXMLWithFile()
         $createMeetingParams->addPresentation('bbb_logo.png', file_get_contents(__DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'fixtures' . DIRECTORY_SEPARATOR . 'bbb_logo.png'));
         $this->assertXmlStringEqualsXmlFile(__DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'fixtures' . DIRECTORY_SEPARATOR . 'presentation_with_embedded_file.xml', $createMeetingParams->getPresentationsAsXML());
     }
+
+    public function testUserCameraCap()
```
Comment (FelixJacobi) [inline tests/unit/Parameters/CreateMeetingParametersTest.php:161]:
```suggestion
    public function testUserCameraCap(): void
```

**Review by FelixJacobi — CHANGES_REQUESTED** [review]
Looks fine to me otherwhise.

**Review by FelixJacobi — APPROVED** [review]

**Pull request merged**